e2e/cypress/integration/interactive_menu/basic_options_spec.js

```diff
@@ -203,6 +203,41 @@ describe('Interactive Menu', () => {
         });
     });
 
+    it('IM21043 - Using up/down arrow keys to make selection', () => {
+        const basicOptions = getMessageMenusPayload({options});
+
+        // # Post an incoming webhook for interactive menu with basic options
+        cy.postIncomingWebhook({url: incomingWebhook.url, data: basicOptions});
+
+        // # Get message attachment from the last post
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#messageAttachmentList_${postId}`).as('messageAttachmentList');
+        });
+
+        cy.get('@messageAttachmentList').within(() => {
+            cy.get('.select-suggestion-container > input').click();
+            cy.get('#suggestionList').should('be.visible');
+
+            // # Hit the down arrow two times
+            cy.get('.select-suggestion-container > input').type('{downarrow}');
+            cy.get('.select-suggestion-container > input').type('{downarrow}');
```

Comment:
Please use suggested alias. 
`.type` can be chained but would prefer to have it as ``.type('{downarrow}{downarrow}')``

Reply:
:+1: